scripts/release-notes/gen_ec_release_notes.py

```diff
@@ -356,10 +392,10 @@ def check_for_new_releases(url_base, release_type, version):
             break
     else:
         rpm_names = ',\n'.join(rpm_list)
-        print(f"WARNING: Did not find {microshift_rpm_name_prefix} in {rpm_names}")
+        logging.warning(f"WARNING: Did not find {microshift_rpm_name_prefix} in {rpm_names}")
```

Comment:
Dont we get the log level automatically with `logging`? Do we need the `WARNING` in the message too?

Reply:
Missed that, fixed!